config/tweaks.json

```diff
@@ -2253,6 +2253,41 @@
       "
     ]
   },
+  "WPFTweaksPowershell7": {
+    "Content": "Replace Default Powershell 5 to Powershell 7",
+    "Description": "This will edit the config file of the Windows Terminal Replacing the Powershell 5 to Powershell 7",
+    "category": "Essential Tweaks",
+    "panel": "1",
+    "Order": "a006_",
+    "InvokeScript": [
+      "
+      if (Test-Path -Path \"$env:ProgramFiles\\PowerShell\\7\") {
+        Write-Host 'Powershell 7 is already installed.'
+      } else {
+        Write-Host 'Installing Powershell 7...'
+        winget install --id Microsoft.PowerShell --silent --accept-source-agreements --accept-package-agreements
```

Comment:
> I think it would be better to utilize the function Install-WinUtilProgramWinget to install Microsoft.Powershell so we keep the different locations in the script where direct calls to winget are used minimal

![image](https://github.com/ChrisTitusTech/winutil/assets/62820836/ff78535d-d431-44f6-b03d-04c216b6ab06)

It's a me problem or I can't Invoke the Script into the File?

![image](https://github.com/ChrisTitusTech/winutil/assets/62820836/3329f4a5-58cc-4147-944e-87525d14a254)